interface/globals.php

```diff
@@ -98,7 +98,7 @@ function undoMagicQuotes($array, $topLevel=true) {
     $tmp = $_GET['site'];
   }
   else {
-    if (!$ignoreAuth) die("Site ID is missing from session data!");
+    if (!isset($ignoreAuth)) die("Site ID is missing from session data!");
```

Comment:
I also confirmed that no error will be thrown if the variable does not exist. So, should use empty here.